tests/integration/proxy/grpcproxy/cluster_test.go

```diff
@@ -90,6 +129,27 @@ func (cts *clusterproxyTestServer) close(t *testing.T) {
 	}
 }
 
+func registDelayDeleteMember(lg *zap.Logger, endpoints []string, addr string, delay time.Duration, ttl int, t *testing.T) chan struct{} {
+	cfg := clientv3.Config{
+		Endpoints:   endpoints,
+		DialTimeout: 5 * time.Second,
+	}
+	client, err := integration2.NewClient(t, cfg)
```

Comment:
Is it possible to pass client from the test without creating a new one?

Reply:
yes, it is